src/index.html

```diff
@@ -1,12 +1,385 @@
-<!doctype html>
-<html lang="en">
+<!DOCTYPE html>
+<html lang="en" class="page">
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" >
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__content content">
+    <header class="header content__header">
+      <a href="#home" class="logo">
+        <img src="./images/logo.png" alt="moyoLogo">
+      </a>
+      <nav class="nav">
```

Comment:
don`t forget to add empty line between sibling tags
```suggestion
      </a>
      
      <nav class="nav">
```